apps/cow-fi/app/(main)/legal/cowswap-terms/page.tsx

```diff
@@ -679,6 +679,44 @@ export default function Page() {
               modification to or deletion of a provision or part-provision under this clause shall not affect the
               validity and enforceability of the rest of these Terms.
             </p>{' '}
+            <h2 id="base-launch">COW Token Giveaway, Base Launch: Terms and Conditions</h2>
+            <h3 id="base-launch-introduction">1. Introduction</h3>
+            <p>By participating in the COW Token Giveaway, you agree to these Terms and Conditions.</p>
+            <h3 id="base-launch-eligibility">2. Eligibility</h3>
+            <p>
+              18+ years old. Not an employee or affiliate of the Organizer. Must have a valid compatible wallet. Must
+              follow CoW Protocol on X. Void where prohibited.
+            </p>
+            <h3 id="base-launch-giveaway-period">3. Giveaway Period</h3>
+            <p>12.12.2024 at the time of the post on the X Platform to December 31, 2023, at 11:59 PM UTC.</p>
```

Comment:
Let's use ISO date format, or the same pattern on both, and the year is wrong in the second part.

```suggestion
            <p>From 2024-12-12 at the time of the post on the X Platform to 2024-12-31, at 11:59 PM UTC.</p>
```